app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource
+
+    def index
+      @conferences_available = Conference.with_roles([:admin, :organizer, :cfp], current_user)
```

Comment:
Unfortunately there is no role named **admin**. Please check again the available roles: organizer, cfp, info_desk, volunteers_coordinator. 
In addition, the User model has an attribute called **is_admin**.